chain/indexer/src/streamer/mod.rs

```diff
@@ -149,34 +163,65 @@ pub async fn build_streamer_message(
             let receipt = if let Some(receipt) = receipt {
                 receipt
             } else {
```

Comment:
nit: [let-else](https://doc.rust-lang.org/rust-by-example/flow_control/let_else.html) is nice

```suggestion
            let Some(receipt) = receipt else {
```